### ISSUE TITLE
app closed - find which notification launched app - popInitialNotification equivalent

### ISSUE THREAD

**Issue by jawadrehman** [oney/react-native-gcm-android#17]
I think this module is 90% complete functionality wise. its just an equivalent of popInitialNotification that seems to be missing.

What I have figured out is that we will need to probably put an extra to the intent in onCreateNotification function of GcmModule.java , and probably create a method to give the value of that.

**Comment by oney** [comment]
Yes, identifying which notification launched app isn't supported yet.
Besides adding info to notification, we should listen intent when launch the app.
In `MainActivity.java`
```java
protected void onCreate(Bundle savedInstanceState) {
    getIntent(); // check if launch from notification
    ...
}
@Override
protected void onNewIntent(Intent intent) {
    super.onNewIntent(intent);
    // send intent to GcmAndroid module
}
```
This is some works to do, I think I may not have time to do it this week.

**Comment by jawadrehman** [comment]
@oney i might have sorted it out - I would like to submit this as a pull request.

**Comment by oney** [comment]
https://github.com/zo0r/react-native-push-notification/blob/b826276e291c0ab011eec80b210d39b63a1c2437/RNPushNotificationAndroid/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java#L50
This module already handles the feature. We can use some code and idea to do it.

**Comment by jawadrehman** [comment]
I have done something similar. But also changed a few lines of your code, since with your code only one notification would show up if the app is not open.

**Comment by oney** [comment]
Can you share codes or create a PR? It will be very useful to me.

**Comment by jawadrehman** [comment]
pull request here. https://github.com/oney/react-native-gcm-android/pull/19
